src/SFA.DAS.AssessorService.ApplyTypes/Roatp/RoatpGatewayConstants.cs

```diff
@@ -11,15 +11,67 @@ public static class RoatpGatewayConstants
         public static class Captions
         {
             public static string OrganisationChecks = "Organisation checks";
+
+            public static string ExperienceAndAccreditation = "Experience and accreditation";
+            public static string OrganisationsCriminalAndComplianceChecks = "Organisation’s criminal and compliance checks";
         }
 
         public static class Headings
         {
+            public static string LegalName = "Legal name check";
+            public static string TradingName = "Trading name check";
             public static string OrganisationStatusCheck = "Organisation status check";
             public static string AddressCheck = "Address check";
             public static string IcoNumber = "Information Commissioner's Office (ICO) registration number check";
             public static string Website = "Website address check";
             public static string OrganisationRisk = "Organisation high risk check";
+
+            public static string OfficeForStudents = "";
+            public static string InitialTeacherTraining = "";
+            public static string Ofsted = "";
+            public static string SubcontractorDeclaration = "Subcontractor declaration check";
```

Comment:
These look like constants to me and should be marked as: public const string

If you need to initialize values at construction, then please mark them as: public static readonly string